src/main/java/io/reactivex/observers/TestObserver.java

```diff
@@ -678,6 +751,65 @@ public final boolean awaitTerminalEvent(long duration, TimeUnit unit) {
     }
 
     /**
+     * Sets the initial fusion mode if the upstream supports fusion.
+     * @param mode the mode to establish, see the {@link QueueDisposable} constants
+     * @return this
+     */
+    public final TestObserver<T> setInitialFusionMode(int mode) {
```

Comment:
How about an @Internal annotation? That one basically declares that while this may be in the public API it is not really part of the public one and subject to change at any time. And in the documentation of the annotation the reasons for having it could be explained.